src/instructlab/training/config.py

```diff
@@ -100,6 +102,11 @@ class TrainingArgs(BaseModel):
     # Either the name of a HuggingFace model or a path to a model saved in HuggingFace format.
     model_path: str
 
+    # Specify the chat template / special tokens for training (default is ibm-generic template/tokens)
+    chat_tmpl_path: str = (
+        os.path.dirname(__file__) + "/chat_templates/ibm_generic_tmpl.py"
```

Comment:
You'll want to us `path.join` here because it handles all of the filepath concatenation well. 

```suggestion
        os.path.join(os.path.dirname(__file__), "/chat_templates/ibm_generic_tmpl.py")
```

Reply:
++ above